### ISSUE TITLE
Integrate with sbt-mode

### ISSUE THREAD

**Issue by hvesalai** [ensime-tmp/ensime-src#2]
Hi,

I have recently published a new sbt-mode for emacs. It is available at [https://github.com/hvesalai/sbt-mode]. It is a complete rewrite, but it was inspired by the sbt code in ensime.

I wanted to write a separate sbt-mode so that it could be maintained and used independently of ensime. However, as there are few elisp hackers with time to hack on the scala-support stuff, I would like to suggest that the ensime project also takes benefit from the new mode. This way the ensime project would have one less component to worry about.

If this sounds like a good idea, I can help integrate the new mode to ensime.

BR, Heikki

**Comment by rajish** [comment]
Good idea! but wait with pull requests to the end of this week, as I have to backport the changes I introduced in the [ensime/ensime](/ensime/ensime) project. You know, the Principles of Reactive Programming course is ongoing...

**Comment by hvesalai** [comment]
Ok, please add a comment here when the time is better.

**Comment by rajish** [comment]
@hvesalai Sorry, that took longer than I thought. I think now it is a good moment to think about a pull request.
